specification/support/resource-manager/readme.md

```diff
@@ -26,18 +26,44 @@ These are the global settings for the Support API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2020-04
+tag: package-preview-2022-09
 ```
 
 
+### Tag: package-preview-2022-09
+
+These settings apply only when `--tag=package-preview-2022-09` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2022-09'
+input-file:
+  - Microsoft.Support/preview/2022-09-01-preview/support.json
+suppressions:
+  - code: RPC-Put-V1-01
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: The path for put operation must be under a subscription and resource group. Justification: Suppressing this rule since path for support ticket resource doesn't contain resource group"
+  - code: RPC-Uri-V1-05
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: The resource name parameter supportTicketName should be defined with a pattern restriction. Justification: We have similar implementation for previous versions, this would be a breaking change, hence suppressing it"
+  - code: RPC-Put-V1-11
+    from: Microsoft.Support/preview/2022-09-01-preview/support.json
+    reason: "Rule: Any Put MUST contain 200 and 201 return codes. Justification: We have similar implementation for previous versions, this would be a breaking change, hence suppressing it"
+  - code: RPC-Patch-V1-03
```

Comment:
>RPC-Patch-V1-03 [](http://example.com/codeflow?start=10&length=15)

This breaks the patch semantics and must be implemented properly.

Reply:
Its not clear which property is being referred here. Can you provide more details and justification?

Reply:
Below are the properties that are referred as errors in private preview PR:

![image](https://github.com/Azure/azure-rest-api-specs/assets/55466801/f75eb399-f0fd-4180-926e-0e16c1e2aa67)


As part of this version, we are doing some minor model updates and added a new API fileworkspace. There were multiple meetings before coming up with design for fileworkspace and implemented accordingly and all other apis are same as previous versions. A detailed explanation was provided to ARM reviewer for the new additions and got an approval.

Reply:
Sorry, I didn't understand this part: "These properties are not supported by our backend systems as per our business logic.". Do you mean, this PATCH API doesn't do anything today? Because if it actually updates status, contactDetails etc. properties in the resource today, the resource modeling needs to be correct for Patch request payload to match the same nested leveling with GET/PUT.